ssas/service/public/api_test.go

```diff
@@ -460,17 +461,25 @@ func (s *APITestSuite) TestTokenEmptyClientIdProduces401() {
 	assert.Nil(s.T(), err)
 }
 
+func GetLogger(logger logrus.FieldLogger) *logrus.Logger {
```

Comment:
can this be imported from the testutils?

Reply:
another good catch - I was testing out this function in a single file and forgot to remove it here. thank you!